index.md

```diff
@@ -4,7 +4,6 @@ layout: lesson
 This Library Carpentry lesson introduces people working in library- and information-related roles to working with data in OpenRefine. At the conclusion of the lesson you will understand what the OpenRefine software does and how to use the OpenRefine software to work with data files.
 
 > ## Prerequisites
->To complete this lesson you will need Firefox web browser and OpenRefine. 
-> If not already installed, you will need to install [Firefox](https://www.mozilla.org/en-US/firefox/new/)
-> and [OpenRefine](http://openrefine.org/download.html). See [Setup](https://librarycarpentry.org/lc-open-refine/setup.html) for more information.
+> To complete this lesson you will need to install [OpenRefine](http://openrefine.org/download.html) and download the file [doaj-article-sample.csv](https://github.com/LibraryCarpentry/lc-open-refine/raw/gh-pages/data/doaj-article-sample.csv).
```

Comment:
I think Ff was originally mentioned to keep the requirements in line with other LC/SC courses. I agree that which browser is used isn't crucial, but I think we need to mention that you need to have a web browser that isn't IE or The Edge installed (I've had workshops where attendees only have IE installed and that's caused problems)

Reply:
Something like this?
> OpenRefine does not support Internet Explorer or Edge. Please use [Firefox](https://www.mozilla.org/firefox/new/), [Chrome](https://www.google.com/chrome/) or [Safari](https://www.apple.com/safari/) instead.

See new commit below.